spec/features/admin/admin_adds_items_spec.rb

```diff
@@ -16,21 +16,20 @@
     click_on "Edit Items"
   end
 
-  xscenario 'can add new item' do
+  scenario 'can add new item' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.